qemu/tests/cfg/win_virtio_driver_install_by_installer.cfg

```diff
@@ -112,6 +110,29 @@
             driver_test_name_balloon = "balloon"
             iozone_cmd_opitons = " -azR -r 64k -n 512M -g 1G -M -I -i 0 -i 1 -b iozone.xls -f %s:\testfile"
             read_rng_cmd = "WIN_UTILS:\\random_%PROCESSOR_ARCHITECTURE%.exe"
+            variants:
+                - with_viomem:
```

Comment:
It's a little weird when getting  all_drivers.with_viomem,  and in the future when there are more drivers support, I think it's hard to extend the variants.
How about not considering the required virtio_win version, just adding description that it was supported from xxx version, because usually this test is run on the new version, even though need to run on the old version, we can easily set `boot_with_viomem = no` to run it.

Reply:
@xiagao yes， but there is another question is that win10.i386 still not support viomem, so seems we still to add such variants. I mean with_viomem and without_viomem.
If we have other new driver we can just add the parameter behind the "viofs " part under @all_drivers 

So base on this do you have any other suggestion?

Reply:
You can use the following code to address your problem.
x86_64:
    boot_with_viomem = yes

Reply:
make sense if we won't consider the rpm version, thanks